exporter/datadogexporter/metrics/utils.go

```diff
@@ -45,20 +46,23 @@ func NewGauge(name string, ts uint64, value float64, tags []string) datadog.Metr
 	return gauge
 }
 
-// RunningMetric creates a metric to report that an exporter is running
-func RunningMetric(exporterType string, timestamp uint64, logger *zap.Logger, cfg *config.Config) []datadog.Metric {
-	runningMetric := []datadog.Metric{
-		NewGauge(fmt.Sprintf("otel.datadog_exporter.%s.running", exporterType), timestamp, float64(1.0), []string{}),
+// DefaultMetrics creates built-in metrics to report that an exporter is running
+func DefaultMetrics(exporterType string, timestamp uint64) []datadog.Metric {
```

Comment:
Probably not for this PR, but wouldn't it make sense to align with other components and use OC for the metrics? This way, people would have a single place to check metrics about all exporters, instead of "all exporters except datadog". Example: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/processor/groupbytraceprocessor/metrics.go

Reply:
Not sure if I understand your point, those metrics are there for observability of the Collector itself, right? This default metric added in #1354 is reported directly to the backend (see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1354#issuecomment-716506278 for an explanation about the usage), it is not something that you can/want to check about the component. Maybe we should clarify that in the function docs?

Reply:
Yes, sorry, I thought this particular one was related to inferring the state of the exporter, which would make it suitable to be placed along with the other exporter's metrics.

Reply:
That makes sense, thanks for clarifying!